README.md

```diff
@@ -1,72 +1,93 @@
 # Metarhia Documentation Generator
 
-## Interface: doc
+[![TravisCI](https://travis-ci.com/metarhia/metadoc.svg?branch=master)](https://travis-ci.com/metarhia/metadoc)
 
-### Introspect interface
+## Installation
 
-`introspect(namespace, text)`
-  - `namespace: `[`<Map>`] hash of interfaces
-  - `text: `[`<string>`] data to parse
+```cmd
```

Comment:
I think `cmd` is supposed to be used for DOS command line (according to [this](https://highlightjs.readthedocs.io/en/latest/css-classes-reference.html#language-names-and-aliases) doc), but here it is used for Linux shell. I think it's better to use `shell` or `sh` here.
```suggestion
```shell
```